wiki/Beatmap_information/en.md

```diff
@@ -0,0 +1,141 @@
+
+The rating spread below the user rating illustrates the proportionate amount of user votes for each number of stars.
+
+## Metadata
+
+### Beatmap description
+
+The beatmap description is a custom field often used for:
+
+- linking ressources such as the background image or hitsound samples
+- commending other users for their help (e.g. guest mappers, modder, storyboarder)
+- trivia facts related to the beatmap (e.g. mapping milestones)
+
+### Keywords
+
+In addition to gameplay-related statistics, every beatmap includes metadata fields to help categorise the beatmap's song. Before submitting a beatmap as part of the [ranking process](/wiki/Beatmap_ranking_procedure), a mapper is required to add this information to the beatmap.
+
+- [Genre](/wiki/Beatmap/Genre_and_language#list-of-genres): The musical or media genre this song mainly uses.
+- [Language](/wiki/Beatmap/Genre_and_language#list-of-languages): In which language the lyrics are, or instrumental if no lyrics are present.
+- [Mapper tags](/wiki/Beatmap/Beatmap_tags#mapper-tags): Useful keywords containing background information about the song.
+- **Source:** The original media the song was produced for, such as a video game or movie.
+
+The nominator section is added as soon as the beatmap is nominated by [beatmap nominators](/wiki/People/Beatmap_Nominators) for becoming ranked.
+
+After the beatmap is ranked, users are able to vote for [user tags](/wiki/Beatmap/Beatmap_tags#user-tags). Upon reaching 5 votes on an user tag, the respective tag will be displayed in the `user tag` segment in this section.
+
+### Success rate display
+
+![](img/success-rate.png "Success rate display")
+
+The success rate display visualises how many players were able to successfully pass the beatmap. When hovering over the percentage value, a tooltip will provide the exact number of players.
+
+Below follows the points of failure, a diagram showing at what points users failed their play. A higher spike means that more players failed at that point.
+
+## Hype train
+
+If a beatmap is WIP or pending, the hype train section is visible where users can [hype](/wiki/Beatmap/Hype) the beatmap to indicate that they want the beatmap to be ranked.
+
+## Rankings
+
+![](img/rankings.png "Global ranking on a beatmap and the top score")
+
+If a beatmap is qualified or ranked, it has access to leaderboards, on which players can compete against each other.
+
+osu! offers the 3 leaderboards `Global Ranking`, `Country Ranking`, and `Friend Ranking`, of which the last two require [osu!supporter](/wiki/osu!supporter) for usage. Selecting one of the tabs at the top of the menu will display the respective leaderboard in the score listing below. Clicking on one or several mod icons in the filter menu allows the user to show solely scores with that exact [mod](/wiki/Gameplay/Game_modifier) combination.
```

Comment:
Bit weird to word this section. We're on the beatmap info page. This is not incorrect, but we want to illustrate what elements can be found in this page, rather than what osu! as a game offers. Plus there are technically more LBS not shown here.
```suggestion
Three leaderboards can be accessed from the beatmap information page, those being `Global Ranking`, `Country Ranking`, and `Friend Ranking`, of which the last two require [osu!supporter](/wiki/osu!supporter) for usage. Selecting one of the tabs at the top of the menu will display the respective leaderboard in the score listing below. Clicking on one or several mod icons in the filter menu allows the user to show solely scores with that exact [mod](/wiki/Gameplay/Game_modifier) combination.
```